src/includes/getting-started-install/android.mdx

```diff
@@ -28,4 +28,6 @@ Sentry's Android SDK depends on [Gson](https://github.com/google/gson) as a tran
 
 If you are using multiple Sentry dependencies, you can add a [bill of materials](/platforms/android/configuration/bill-of-materials) to avoid specifying the version of each dependency.
 
+If you are using Kotlin and coroutines in your Android project, you should read [Using Sentry SDK with Coroutines](/platforms/android/enriching-events/scopes/#kotlin-coroutines)
```

Comment:
This should not be here, since it only makes sense to use `SentryContext` if you have the concept of Local scopes, which is not the case for Android.

Reply:
That totally makes sense, and I agree. The existing documentation both tells you how to use them and that you can't, so I erred on the side of improving what was there. Specifically, I removed directions on how to use them while leaving the mention that they can't be used as I wasn't completely comfortable with removing it entirely without a review. I can remove the mention entirely if that is preferred. 


| existing | this PR |
|---------|--------|
| ![image](https://user-images.githubusercontent.com/2702309/156109179-fb12e859-644d-49f8-96a1-02409d177114.png) | ![image](https://user-images.githubusercontent.com/2702309/156109282-27390602-7ae4-4f6d-83b6-43fcabb8bca7.png) |

Reply:
Yep, that would be ideal, I'd rather keep only for Java as it is right now https://docs.sentry.io/platforms/java/enriching-events/scopes/#kotlin-coroutines

Reply:
@marandaneto, finally got back to this, done in 014eaa22

Reply:
Not sure if its not up to date (Vercel didn't deploy changes?), but I can still see it for Android https://sentry-docs-git-fork-designedbyz-update-scopes.sentry.dev/platforms/android/enriching-events/scopes/#kotlin-coroutines